testing/e2e_test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
The script has "e2e" in the name and then calls "integration_test.sh". What's the difference between "e2e" and "integration" tests in this context? Are we just using two words for the same thing? If so, let's not :)

Also, this whole file seems to be just `chmod uog+x $(dirname $0)/integration_test.sh && $(dirname $0)/integration_test.sh`?